automatic/glmixer/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,19 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageArgs = @{
+  packageName            = 'glmixer'
+  fileType               = 'exe'
+  url                    = 'https://sourceforge.net/projects/glmixer/files/Windows%20Binary/GLMixer_1.5-1329_Windows.exe'
```

Comment:
I mean, I removed the url but `update.ps1` just fills it again. If I remove the replace from `update.ps1`, then I get that error.